stories/guidelines/i18n/I18n.stories.mdx

```diff
@@ -81,38 +106,135 @@ type I18nResource = Readonly<{
 }>
 ```
 
+#### The 3 ways to implement translations explained step by step
+
+The `LanguageSwitcher`component comes with its own translations which are stored in the `LanguageSwitcher_en.json` file.  
```

Comment:
As this part of the documentation is fully agnostic about other UI components, we could rather mention a fake component as example (more neutral), what do you think ? 
```suggestion
Given an `AwesomeComponent` that comes with its own translations which are stored in the `AwesomeComponent_en.json` file.  
```

Reply:
Absolutely right, thanks !